### DISCUSSION TITLE
Specify input image CRS

### DISCUSSION THREAD

**Discussion by mapper82** [azavea/raster-vision#1718] [Q&A]
Awesome tool, thanks for the hard work here!

Im dealing with some input rasters that do not have a defined coordinate system. The crs of these rasters is actually `EPSG:6527`, its just not part of the meta for each image. (verified missing crs using gdalinfo)

Is there a way to pass the crs into the `RasterioSourceConfig` or somewhere else? My code loosely follow the [tiny_spacenet](https://github.com/azavea/raster-vision/blob/master/rastervision_pytorch_backend/rastervision/pytorch_backend/examples/tiny_spacenet.py) example. Thanks in advance.

**Comment by AdeelH** [comment]
I think something like an `override_crs` argument in `RasterioSource[Config]` makes sense for such cases.

It might be simpler to just use GDAL to set the CRS metadata on the rasters though. I haven't done this before, but this seems like a way to do it: https://gis.stackexchange.com/a/72452/181526.
